types/magicmirror-module/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for non-npm package magicmirror-module 2.16
+// Project: https://magicmirror.builders/
+// Definitions by: Jalibu <https://github.com/jalibu>
+//                 MichMich <https://github.com/MichMich>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
```suggestion
// TypeScript Version: 3.7
```
3.0 is no longer enforced and tested on DT. You can skip this header completely (3.7 is the lowest tested version here now)